src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -31,23 +40,31 @@ import Timer from "../../components/Timer/Timer";
 import Modal from "react-modal";
 import Card from "../Card/Card";
 import WalletProviderList from "../WalletProviderList/WalletProviderList";
+import TokenSelection from "../../components/TokenSelection/TokenSelection";
 
 const floatRegExp = new RegExp("^([0-9])*[.,]?([0-9])*$");
 
 const SwapWidget = () => {
   const [senderToken, setSenderToken] = useState<string>();
   const [signerToken, setSignerToken] = useState<string>();
-  const [senderAmount, setSenderAmount] = useState("0.01");
+  const [senderAmount, setSenderAmount] = useState<string>("0.01");
```

Comment:
Use number type for senderAmount

Reply:
I believe we've kept this as string because `getSignerSideOrder` is expecting a string

Reply:
Ah ok then it can stay like this.